wandb/util.py

```diff
@@ -1014,11 +1014,7 @@ def image_from_docker_args(args: List[str]) -> Optional[str]:
 
 
 def load_yaml(file: Any) -> Any:
-    """If pyyaml > 5.1 use full_load to avoid warning"""
```

Comment:
Open question: what warning was this? And do we actually use the functionality of full_load?

Reply:
It must have been a deprecation warning like shown here https://github.com/yaml/pyyaml/issues/265